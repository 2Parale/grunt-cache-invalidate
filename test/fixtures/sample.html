<!DOCTYPE html>
<html>
<head>
    <title>Hello World!</title>
    <script type="text/javascript" src="sample1.js#grunt-cache-invalidate"></script>
    <script type="text/javascript" src="sample2.js#grunt-cache-invalidate"></script>
    <script type="text/javascript" src="sub/sample2.js#grunt-cache-invalidate"></script>
    <script type="text/javascript" src="sub/sample3.js#grunt-cache-invalidate"></script>
</head>
<body>

</body>
</html>