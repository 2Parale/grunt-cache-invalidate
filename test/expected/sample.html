<!DOCTYPE html>
<html>
<head>
    <title>Hello World!</title>
    <script type="text/javascript" src="sample1.5340dc1df.js"></script>
    <script type="text/javascript" src="sample2.e7d97adc3.js"></script>
    <script type="text/javascript" src="sub/sample2.another.e7d97adc3.js"></script>
    <script type="text/javascript" src="sub/sample3.85a5d055f.js"></script>
</head>
<body>

</body>
</html>